app/models/answer.rb

```diff
@@ -13,4 +13,8 @@ class Answer < ActiveRecord::Base
   validates :question, presence: true
   validates :content, presence: true
 
+  def temp_id!
+    self.temp_id ||= SecureRandom.hex(8)
+  end
```

Comment:
This supports testing -- helps simulate how the FE uses `temp_id` to connect objects together in the JSON before they have been persisted to the DB.  I'm open to other suggestions if this seems awkward.